hw1_2/.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove *.ipr file from the repository
Remove *.iws file from the repository
Remove .idea file from the repository
Remove target folder from the repository
Put pom.xml to root of the repository with project